samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/ScanActivity.kt

```diff
@@ -0,0 +1,48 @@
+package org.mozilla.samples.fxa
+
+import android.Manifest
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.v4.app.ActivityCompat
+import android.support.v4.content.ContextCompat
+import android.util.Log
+import com.google.zxing.Result
+import me.dm7.barcodescanner.zxing.ZXingScannerView
+
+class ScanActivity : AppCompatActivity(), ZXingScannerView.ResultHandler  {
+
+    private var mScannerView: ZXingScannerView? = null
```

Comment:
Let's make this a `lateinit` var, then we don't need the `!!` or `?` operators every time.